pytorch_lightning/core/saving.py

```diff
@@ -372,10 +373,20 @@ def save_hparams_to_yaml(config_yaml, hparams: Union[dict, Namespace]) -> None:
                     OmegaConf.save(OmegaConf.create(hparams), fp, resolve=True)
```

Comment:
Hey @Borda, I think there might be a bug there. I need to create a small gist to make sure of it.
We should resolve the config before if hydra is using interpolated fields.

Reply:
this does not seems to related to this PR...